TestScripts/profile_validate_testscript.json

```diff
@@ -0,0 +1,146 @@
+    {
+      "code": {
+        "system": "http://terminology.hl7.org/CodeSystem/usage-context-type",
+        "code": "focus"
+      },
+      "valueCodeableConcept": {
+        "coding": [
+          {
+            "system": "http://terminology.hl7.org/CodeSystem/variant-state",
+            "code": "positive"
+          }
+        ]
+      }
+    }
+  ],
+  "jurisdiction": [
+    {
+      "coding": [
+        {
+          "system": "urn:iso:std:iso:3166",
+          "code": "US",
+          "display": "United States of America (the)"
+        }
+      ]
+    }
+  ],
+  "purpose": "Patient Conditional Create (Update), Read and Delete Operations",
+  "copyright": "© HL7.org 2011+",
+  "metadata": {
+    "link": [
+      {
+        "url": "http://hl7.org/fhir/patient.html",
+        "description": "Demographics and other administrative information about an individual or animal receiving care or other health-related services."
+      }
+    ],
+    "capability": [
+      {
+        "required": true,
+        "validated": false,
+        "description": "Patient Update, Read and Delete Operations",
+        "link": [
+          "http://hl7.org/fhir/http.html#delete",
+          "http://hl7.org/fhir/http.html#read",
+          "http://hl7.org/fhir/http.html#update"
+        ],
+        "capabilities": "CapabilityStatement/example"
+      }
+    ]
+  },
+  "fixture": [
+    {
+      "id": "fixture-patient-create",
+      "autocreate": false,
+      "autodelete": false,
+      "resource": {
+        "reference": "fixtures/Patient/patient_example_uscore.json",
+        "display": "Peter Chalmers"
+      }
+    }
+  ],
+  "profile": [
+    {
+      "id": "patient-uscore-profile",
+      "reference": "http://hl7.org/fhir/us/core/StructureDefinition/us-core-patient"
+    }
+
+  ],
+  "variable": [
+    {
+      "name": "createResourceId",
+      "path": "Patient/id",
+      "sourceId": "fixture-patient-create"
+    }
+  ],
+  "test": [
+    {
+      "id": "01-ReadPatient",
+      "name": "Read Patient",
+      "description": "Read a Patient and validate response.",
+      "action": [
+        {
+          "operation": {
+            "type": {
+              "system": "http://terminology.hl7.org/CodeSystem/testscript-operation-codes",
+              "code": "read"
+            },
+            "resource": "Patient",
+            "description": "Read the patient resource on the test server using the id from fixture-patient-create. Prevent URL encoding of the request.",
+            "encodeRequestUrl": false,
+            "responseId": "fixture-patient-read",
+            "targetId": "fixture-patient-create"
+          }
+        },
+        {
+          "assert": {
+            "label": "01-ReadPatientOK",
+            "description": "Confirm that the returned HTTP status is 200(OK).",
+            "direction": "response",
+            "response": "okay",
+            "warningOnly": false
+          }
+        },
+        {
+          "assert": {
```

Comment:
remove all but this assert from the test since they aren't relevant to what is being tested

Reply:
Will do

Reply:
Done:  https://github.com/fhir-crucible/testscript-engine/pull/70/commits/40dc24f7f6a7fc4b24ebc0c2f0aa5f896a3572bf